docs/reference/article_fragment.rst

```diff
@@ -0,0 +1,13 @@
+Article's Fragment
+==================
+
+This feature move the removed fragment input's in another DOM and thereafter
+this allows the cancellations of removed fragments.
+After the deletion of a fragment, each deleted fragment item in the fragment list
+contain a reset button (to undo the deletion) and the fragment title. This action triggers undo of
+the deletion of the fragment so the moved fragment input's (and elements) are moving in their original place.
+
+.. note::
+    This feature fix too a bug of the removal of predefined fragments in the creation of new article.
+
+
```

Comment:
this file should only concern fragment deletion so should be named delete_fragment.rst IMO. It has to be referenced in index.rst file (will fix a part of the build) (and you should rebase ;) will fix the second part of the build 
 i mean flintCI part)